WebRoot/skins/_base/base4/skin.css

```diff
@@ -832,4 +832,35 @@ div.ImgTagStack {
 
 .DwtTreeItem .svg-icon {
     fill: @TreeItemTextColor@
+}
+
+/*
+* DwtDialog Button styles
+*/
+.ZDialogButton {
+    margin-right: 8px; 
+}
+
+.ZDialogButton.ZPrimaryButton {
+    margin: 0;
+}
+
+.ZDialogButton .ZButtonTable .ZWidgetTitle {
+    color: @AltC@;
+    letter-spacing: 1px;
+}
+
+.ZPrimaryButton .ZButtonTable {
+    background: @AltC@;
+}
+
+.ZPrimaryButton.ZHover    .ZButtonTable,
+.ZPrimaryButton.ZFocused  .ZButtonTable,
+.ZPrimaryButton.ZActive   .ZButtonTable,
+.ZPrimaryButton.ZSelected .ZButtonTable {
+    background: @darken(AltC,30)@;
```

Comment:
Moved to `zm.css` https://github.com/Zimbra/zm-web-client/pull/5/files#diff-33f9c57f22cd743824b2918132c5d73bR6289